### ISSUE TITLE
Readable error message when I enter the wrong password

### ISSUE THREAD

**Issue by mawenzel** [zalando-stups/mai#1]
when I want start mai with e.g. mai create stups, then I have to enter my credentials and an url. If I enter a wrong password, then I get an error message. But this message did not show me that my password is incorrect. 

```
  EXCEPTION OCCURRED: 
Traceback (most recent call last):
  File "/usr/local/bin/mai", line 9, in <module>
    load_entry_point('stups-mai==0.5', 'console_scripts', 'mai')()
  File "/usr/local/lib/python3.4/site-packages/mai/cli.py", line 126, in main
    cli()
  File "/usr/local/lib/python3.4/site-packages/click/core.py", line 664, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.4/site-packages/click/core.py", line 644, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.4/site-packages/click/core.py", line 991, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.4/site-packages/click/core.py", line 837, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.4/site-packages/click/core.py", line 464, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.4/site-packages/click/decorators.py", line 26, in new_func
    return ctx.invoke(f, ctx.obj, *args[1:], **kwargs)
  File "/usr/local/lib/python3.4/site-packages/click/core.py", line 464, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.4/site-packages/mai/cli.py", line 60, in create
    saml_xml, roles = authenticate(url, user, saml_password)
  File "/usr/local/lib/python3.4/site-packages/aws_saml_login/saml.py", line 133, in authenticate
    raise AuthenticationFailed()
aws_saml_login.saml.AuthenticationFailed
```

**Comment by hjacobs** [comment]
This should be fixed in the latest version (0.6):

```
sudo pip3 install --upgrade stups-mai
```

**Comment by mawenzel** [comment]
Is fixed. thx
